### PR TITLE
Vectorize convert angles node

### PR DESCRIPTION
**Pull request by harisreedhar**

### CODE REVIEW

File: animation_nodes/math/number.pyx [hunk 1]
```diff
@@ -87,3 +87,9 @@ cdef double lerp(double x, double y, double p):
 
 cdef float lerpFloat(float x, float y, float p):
     return (1.0 - p) * x + p * y
+
+cdef double degToRad(double value):
+    return value * PI / 180    
+
+cdef double radToDeg(double value):
+    return value * 180 / PI
```
Comment (OmarEmaraDev) [inline animation_nodes/math/number.pyx:95]: Use full names. `degreeToRadian` `radianToDegree`.

**Review by OmarEmaraDev — CHANGES_REQUESTED** [review]

**Comment by OmarEmaraDev** [comment]
Don't forget to update the CHANGELOG.

**Comment by harisreedhar** [comment]
> Don't forget to update the CHANGELOG.

Done

File: animation_nodes/math/number.pyx (hunk 1 above)
Comment (harisreedhar) [reply animation_nodes/math/number.pyx]: > Use full names. `degreeToRadian` `radianToDegree`.

Done

File: animation_nodes/nodes/number/c_utils.pyx [hunk 2]
```diff
@@ -146,3 +147,14 @@ def compareNumbers_GreaterThanOrEqual(VirtualDoubleList a, VirtualDoubleList b,
...
+    cdef Py_ssize_t i
+    cdef Py_ssize_t amount = values.length
+    cdef DoubleList result = DoubleList(length = amount)
+    if mode == 'DEGREE_TO_RADIAN':
+        for i in range(amount):
+            result.data[i] = degToRad(values.data[i])
+    elif mode == 'RADIAN_TO_DEGREE':
+        for i in range(amount):
+            result.data[i] = radToDeg(values.data[i])
+    return result
```
[the remark this answers is not in the document]
Comment (harisreedhar) [reply animation_nodes/nodes/number/c_utils.pyx]: Done

**Pull request merged**